pkg/config/parameter/file/file.go

```diff
@@ -93,14 +94,18 @@ func parseFileValueParameter(context parameter.ParameterParserContext) (paramete
 		return nil, parameter.NewParameterParserError(context, "missing filesystem handle to load parameter")
 	}
 
-	path, ok := context.Value["path"]
+	p, ok := context.Value["path"]
 	if !ok {
 		return nil, parameter.NewParameterParserError(context, "missing property `path`")
 	}
 
+	path := strings.ToString(p)
```

Comment:
I think we should perform some proper validation here and type-cast it. Imo it should error if it's not a string.

Reply:
Line #130 also no longer needs the `strings.ToString(path)`, right?